Python/Tests/DebuggerTests/DebuggerTests.cs

```diff
@@ -1968,6 +1970,8 @@ private enum ExceptionMode {
                             attached.Set();
                             breakpointHit.Set();
                         }
+
+                        continueEvent.WaitOne(1000);

```

Comment:
Actually this is wrong altogether, it shouldn't wait in the debugger callback because that blocks the debugger thread and prevents the execution result from being received - so it was _always_ timing out (but waiting still broke the race condition so I didn't notice). I'll rewrite this to wait on the main thread, which can then properly assert.